client/components/post-list-fetcher/index.jsx

```diff
@@ -143,7 +151,7 @@ PostListFetcher = React.createClass( {
 	},
 
 	onPostsChange: function() {
-		this.setState( getPostsState( this.props ) );
```

Comment:
correct if you look at line 56 you will see `getPostsState` was not using the props at all.  Here we need the `postListStoreId` to work against the correct instance of the store.